### PR TITLE
fix control panel role principal and permissions

### PR DESCRIPTION
**Pull request by sh31**

### CODE REVIEW

File: modules/control_panel/role.tf [hunk 1]
```diff
@@ -42,6 +42,7 @@ resource "aws_iam_policy" "control_panel" {
         "s3:CreateBucket",
         "s3:PutBucketLogging",
         "s3:PutBucketTagging",
+        "s3:PutBucketPublicAccessBlock",
```
Comment (xoen) [inline modules/control_panel/role.tf]: We'll need this in the old terraform stuff too ☺️

**Comment by xoen** [comment]
@sh31 can this be merged or it would cause problems?

**Comment by sh31** [comment]
Merging

File: modules/control_panel/role.tf (hunk 1 above)
Comment (xoen) [reply modules/control_panel/role.tf]: FYI: [Andy added this `s3:PutBucketPublicAccessBlock ` permission in the other repo](https://github.com/ministryofjustice/analytics-platform-ops/pull/293) so no need to do anything there 🏆

**Comment by xoen** [comment]
Brilliant, thanks @sh31 ❤️

**Pull request merged**